homeassistant/components/ecobee/climate.py

```diff
@@ -163,15 +167,15 @@ def resume_program_set_service(service):
 
             thermostat.schedule_update_ha_state(True)
 
-    hass.services.register(
-        DOMAIN,
+    hass.services.async_register(
+        CLIMATE_DOMAIN,
         SERVICE_SET_FAN_MIN_ON_TIME,
         fan_min_on_time_set_service,
         schema=SET_FAN_MIN_ON_TIME_SCHEMA,
     )
 
-    hass.services.register(
-        DOMAIN,
+    hass.services.async_register(
+        CLIMATE_DOMAIN,
```

Comment:
We shouldn't register custom services under the platform domain but under the integration domain, ie ecobee.

Reply:
Sounds good to me.